runtime/onert/core/src/backend/cpu_common/Tensor.cc

```diff
@@ -23,10 +23,13 @@ namespace onert
 {
 namespace backend
 {
+// `dynamic_cast` not working across library boundaries on NDK
+// With this as a key function, `dynamic_cast` works across dl
+IPortableTensor::~IPortableTensor() {}
```

Comment:
I am sure that this works as a keyfunction successfully but I am worried if defining super class's function in child class like this.. @wateret